From henne@nachtwindheim.de Tue Aug 15 01:57:58 2006
Message-ID: <44E18C6C.2080902@nachtwindheim.de>
Date: Tue, 15 Aug 2006 10:57:16 +0200
From: Henne <henne@nachtwindheim.de>
To: gregkh@suse.de
Subject: PCI: kerneldoc correction in pci-driver

From: Henrik Kretzschmar <henne@nachtwindheim.de>

Removes an unused kerneldoc entry from pci_match_device and
put the others into correct order.

Signed-off-by: Henrik Kretzschmar <henne@nachtwindheim.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/pci/pci-driver.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- gregkh-2.6.orig/drivers/pci/pci-driver.c
+++ gregkh-2.6/drivers/pci/pci-driver.c
@@ -149,9 +149,8 @@ const struct pci_device_id *pci_match_id
 /**
  * pci_match_device - Tell if a PCI device structure has a matching
  *                    PCI device id structure
- * @ids: array of PCI device id structures to search in
- * @dev: the PCI device structure to match against
  * @drv: the PCI driver to match against
+ * @dev: the PCI device structure to match against
  *
  * Used by a driver to check whether a PCI device present in the
  * system is in its list of supported devices.  Returns the matching
